src/app/browse-by/browse-by-metadata-page/browse-by-metadata-page.component.ts

```diff
@@ -122,19 +122,19 @@ export class BrowseByMetadataPageComponent implements OnInit, OnDestroy {
   fetchThumbnails: boolean;
 
   public constructor(protected route: ActivatedRoute,
-                     protected browseService: BrowseService,
-                     protected dsoService: DSpaceObjectDataService,
-                     protected paginationService: PaginationService,
-                     protected router: Router,
-                     @Inject(APP_CONFIG) public appConfig: AppConfig) {
+    protected browseService: BrowseService,
```

Comment:
@CrisGuzmanS : Would it be possible to put the alignment back to how it was in these files?  It appears your IDE or editor has changed these files so that the parameters to methods are no longer aligned... It happened in all the methods of this file, so it was probably an automated change.  

But, it makes this PR appear much large then it actually is... plus, the code is harder to read when these params are no longer aligned.

Reply:
@tdonohue for sure! al already did it 👍